source/includes/administration/_resource_commitments.md

```diff
@@ -0,0 +1,343 @@
+## Resource commitments
+
+Resource commitments allow you to set specific commitment levels on cloud resources `(compute, network & storage)` avaialble on a [service-connection](#administration-service-connections). This assures commitment to a specific set of resources on a fixed monthly price. Any additional resource usage will be billed on the utility price defined by the CMC [pricing](#administration-pricing). Commitments can either be restricted to a specific date range or be un-restricted with no end date. Whilst the commitment price is decided by the pre-defined [pricing](#administration-pricing), various discounts can be given on the resource utility price. Resource commitments allow Cloud-MC users to receive services at fixed discounted rates whilst enabling additional usage at utility cost.
+
+<!-------------------- LIST RESOURCE COMMITMENTS -------------------->
+
+### List resource commitments
+
+`GET /resource_commitments`
+
+```shell
+# Retrieve visible environments
+curl "https://cloudmc_endpoint/v2/resource_commitments" \
+   -H "MC-Api-Key: your_api_key"
+
+# Response body example
+```
+```json
+{
+  "data": [{
+    "id": "f5dc7647-71e6-4b9b-998a-c0e4d38bf2e6",
+    "displayName": "CRB-SYSTEM-1",
+    "creationDate": "2019-05-24T18:02:01.000Z",
+    "startDate": "2019-05-16T00:00:00Z",
+    "endDate": "2019-05-24T00:00:00Z",
+    "pricingMethod": "FIXED_MONTHLY",
+    "currency": "CAD",
+    "monthlyPrice": 20.0,
+    "effectiveDiscount": 92.39,
+    "billableHoursPerMonth": 730,
+    "billingDay": 16,
+    "resourceCommitmentDetails": [
+      {
+        "id": "08125b3c-3b25-4585-910b-c1a0dc53010a",
+        "name": "CustomDoF",
+        "primaryType": "6",
+        "secondaryType": "CustomDoF",
+        "commitment": 0,
+        "discountPercent": 0,
+        "unit": "gigabyte",
+        "labelKey": "cloudstack.volume"
+      },
+      {
+        "id": "0bf1a48d-f807-4821-8247-2eb9a3128091",
+        "primaryType": "3",
+        "commitment": 0,
+        "discountPercent": 0,
+        "unit": "unit",
+        "labelKey": "cloudstack.ipAddress"
+      },
+      { ... },
+      { ... },
+      { ... }
+    ],
+    "serviceConnection": {
+      "name": "dev1-cloudstack",
+      "id": "cb511a38-eab5-4a2e-969b-04e129c98415",
+      "type": "cloudstack"
+    },
+    "organization": {
+      "name": "System",
+      "id": "52fd201e-aa82-4a27-86b3-ea9650a7fb1e"
+    },
+    "version": 1,
+    "deleted": false
+  }, 
+  {
+    "id": "51c01cfe-f0d5-4a3d-841d-35ec2b7df9cc",
+    "displayName": "CRB-SYSTEM-2",
+    ....
+  }]
+}
+```
+List all the resource commitments that you have access to. You can only view resource commitments if you have been assingned either the `View usage` or the `Manage commitments` permission. If you have `View usage` access then you will only be able to see the commitments specific to your organization. Having the `Manage commitments` permission lets you see all resource commitments in the system.
```

Comment:
assingned  => assigned

Reply:
Fixed in 738496526a7aa1f45bc86f95f7c32cc0e4ed3320